src/test/java/org/springframework/data/tarantool/repository/support/RepositoryIntegrationTest.java

```diff
@@ -137,4 +145,65 @@ public void testBatchSave() {
         List<Book> savedBooks = bookRepository.batchSave(Arrays.asList(book1, book2));
         assertTrue(savedBooks.size() > 0);
     }
+
+    @Test
+    public void testInvocationInternalFunctionWithoutTupleAnnotationShouldReturnResponseAsObject() {
+        SampleUser expected = SampleUser.builder()
+                .name("Vasya")
+                .lastName("Vasiliev")
+                .build();
+
+        SampleUser sampleUser = sampleUserRepository.findByName("test");
+        assertThat(sampleUser).isEqualTo(expected);
+    }
+
+    @Test
+    public void testSaveEntityWithoutTupleAnnotationShouldSaveAndReturnResponse() {
```

Comment:
I suggest to write comments in style of Given, When, Then.
Here it will be like:
// When Tuple is specified on Repository
// Then implicit save returns a valid object

OR rename method to 

testImplicitSaveWhenTupleIsOnRepository